lms/static/sass/_header.scss

```diff
@@ -0,0 +1,365 @@
+      }
+    }
+
+    // Display and style all mobie menu elements
+    .nav-item,
+    .dropdown-nav-item {
+      display: inline-block;
+      width: 100%;
+      border: none;
+      padding: $baseline/5 0 0;
+      margin: $baseline*0.75 $baseline 0;
+      color: theme-color("primary");
+      border: none;
+      text-decoration: none;
+      background-color: transparent;
+      font-weight: normal;
+      font-size: 20px;
+
+      a {
+        text-decoration: none;
+
+        .active, // Indicates if we are currently on one of the pages
+        &:hover{
+          color: theme-color("dark");
+        }
+      }
+
+      &:hover {
+        color: theme-color("dark");
+        font-weight: 600;
+      }
+    }
+
+    // Custom focus behavior for tabbing through elements (links, dropdown links, and buttons)
+    .nav-item a:focus,
+    .dropdown-nav-item a:focus,
+    a.nav-item:focus {
+      outline: none;
+      color: theme-color("dark");
+      font-weight: 600;
+    }
+
+
+    // Custom styling to ensure dropdown elements match other nav elements in collapsed mode
+    .dropdown-user-menu {
+      display:inline !important;
+
+      .dropdown-nav-item {
+        @include margin-left(0);
+      }
+
+      a:first-child{
+        .dropdown-nav-item {
+          margin-top: 0;
+        }
+      }
+    }
+
+    // Display the menu icon and allow for transition to an X on click
+    .hamburger-menu {
+      @include right(22px);
+      position: absolute;
+      top: 24px;
+      width: 30px;
+      height: 20px;
+      -webkit-transform: rotate(0deg);
+      -moz-transform: rotate(0deg);
+      -o-transform: rotate(0deg);
+      transform: rotate(0deg);
+      -webkit-transition: .5s ease-in-out;
+      -moz-transition: .5s ease-in-out;
+      -o-transition: .5s ease-in-out;
+      transition: .5s ease-in-out;
+      cursor: pointer;
+      outline: none;
+
+      &:focus,
+      &:hover {
+        span {
+          background: theme-color("dark");
+          height: 3px;
+        }
+      }
+
+      span {
+        display: block;
+        position: absolute;
+        height: 2px;
+        width: 100%;
+        background: theme-color("secondary");
+        border-radius: 9px;
+        opacity: 1;
+        left: 0;
+        -webkit-transform: rotate(0deg);
+        -moz-transform: rotate(0deg);
+        -o-transform: rotate(0deg);
+        transform: rotate(0deg);
+        -webkit-transition: .25s ease-in-out;
+        -moz-transition: .25s ease-in-out;
+        -o-transition: .25s ease-in-out;
+        transition: .25s ease-in-out;
+
+        &:nth-child(1) {
+          top: 0px;
+        }
+
+        &:nth-child(2), &:nth-child(3) {
+          top: 6px;
+        }
+
+        &:nth-child(4) {
+          top: 12px;
+        }
+      }
+
+      &.open span {
+        &:nth-child(1) {
+          top: 18px;
+          width: 0%;
+          left: 50%;
+        }
+
+        &:nth-child(2) {
+          -webkit-transform: rotate(45deg);
```

Comment:
Are all four of these needed now for modern browsers? My suspicion is that you could just use a single line instead of all four of these.